Apps/Examples/Examples/All Examples/GlobeFlyToExample.swift

```diff
@@ -0,0 +1,93 @@
+import Foundation
+import UIKit
+@_spi(Experimental) import MapboxMaps
+import CoreLocation
+
+class GlobeFlyToExample: UIViewController, ExampleProtocol {
+    internal var mapView: MapView!
+    internal var currentProjection = StyleProjection(name: .globe)
+    internal var isAtStart = true
+    var instuctionLabel = UILabel(frame: CGRect.zero)
+
+    private var CAMERA_START = CameraOptions(
+        center: CLLocationCoordinate2D(latitude: 36, longitude: 80),
+        zoom: 1.0,
+        bearing: 0,
+        pitch: 0)
+
+    private var CAMERA_END = CameraOptions(
+        center: CLLocationCoordinate2D(latitude: 46.58842, longitude: 8.11862),
+        zoom: 12.5,
+        bearing: 130.0,
+        pitch: 75.0)
+
+    override public func viewDidLoad() {
+        super.viewDidLoad()
+
+        mapView = MapView(frame: view.bounds, mapInitOptions: .init(styleURI: .satelliteStreets))
+        mapView.autoresizingMask = [.flexibleWidth, .flexibleHeight]
+        mapView.mapboxMap.setCamera(to: .init(center: CLLocationCoordinate2D(latitude: 40, longitude: -78), zoom: 1.0))
+
+        mapView.mapboxMap.onNext(event: .styleLoaded) { _ in
+            try! self.mapView.mapboxMap.style.setProjection(self.currentProjection)
```

Comment:
Set the projection directly in the `viewDidLoad` - no need to wait for style to load.